config/locales/account/en.yml

```diff
@@ -109,4 +109,5 @@ en:
     security:
       link: Learn more at the Help Center
       text: Your profile information is locked for your security.
+    terms: I agree to login.gov Terms of Use and the Security and Privacy Practices.
```

Comment:
In #5040, we embed a link to the rules in the text. Should we be doing that here? Otherwise, it kinda feels like we're asking the user to agree to something without giving them a chance to read what they're agreeing to. Or will we update the text "Security Practices and Privacy Act Statement" to also include "Rules of Use" ?

Reply:
This was part of the discrepancies I posted in slack.  It appears they want the link "Security Practices and Privacy Act Statement" to go to the new "Rules of Use".  There are other tabs there for the security practices and privacy act statement.  It does seem odd.